### ISSUE TITLE
Mekanism v9 : Quantum Entangloporter stopped working

### ISSUE THREAD

**Issue by Giovanni-NL** [BevoTechPack/BevoTechPack#998]
Since release BetaS of this pack the Mekanism Quantum Entangloporter stopped working.  This seems to be a known issue, and can be fixed by updating a config file and some mods available in this pack. (EnderIO and MFR to name just 2)

For more details, please see this issue on the Mekanism GitHub pages.
https://github.com/aidancbrady/Mekanism/issues/3273#issuecomment-222337658

**Comment by b0bst3r** [comment]
I'm loathed to add EIO 426 because of this issue report SleepyTrousers/EnderIO#3333

EIO addons is not in the pack you must have added that.

MFR is already on a newer version in BetaS

Can do ExtraCells but not EIO until they fix the above issue.

**Comment by MRuth** [comment]
I see that my comment on the Mekanism Quantum Entangleporter was referenced.  If it would be possible for others to test this out and report their findings.  I am running a very large modpack (a modified version of Resonant Rise) with around 250+ mods, and even after posting what I thought might be a possible fix for the entangleporter issue, I am still getting some instances in which the entangleporter does not connect and resume transfer after a world reload, and have not been able to specifically pin the issue down yet.

**Comment by Giovanni-NL** [comment]
Just to clear this up: I'm using the default settings of Bevo's pack, no mods have been added by me, nor do I wish for extra mods to be added.  The reference to EIO addons is on MRuth's modified version of Resonant Rise, and I cannot comment on that. 

If EIO 426 does indeed introduce new problems, then perhaps it's best to wait for a newer version of EIO before upgrading.  

There is a workaround, and that is to add a new channel to the Quantum Entangloporter when this problem occurs.  The new channel will work, but the old channel will stay broken.  Going back to the old channel will not transfer gasses (in my particular setup), and then going to newly created channel again will fix that.  This would have to be repeated every time a QE stops working (for whatever reason).  Clearly this is not an optimal solution, especially if there are frequent server restarts.

**Comment by MRuth** [comment]
What is confusing me, is that I had downloaded a brand new version of Bevo's Tech Pack with all mods enabled, created a world and tested the entangleporters.  After a world reload, the entangleporters would not connect and transfer.  However once I disabled pretty much every mod except for Mekanism, forge, and Chicken Chunks and CodeChicken Core, a new world was created and the entangleporters worked both during gameplay and after a world reload.  I then spent time enabling all other mods in batches until all mods had been re-enabled.  Each time I would test the entangleporters after a world reload, and they worked.  After the last batch had been enabled, I not only tested the world reload on an existing world, but also created a new world from scratch to test, and the entangleporters worked both during gameplay and after a world reload, which should not be happening as they were broken when the pack was first played with the same mods loaded...
